server/schema/customer/res.js

```diff
@@ -37,15 +37,17 @@ const resolvers = {
     async genCustomerPaymentId(_, {input}, {loggedInUser}) {
       _auth(loggedInUser)
       let user = await Customer.findById(input)
-      if (user) return jwt.sign({id: input}, process.env.JWT_SECRET, {expiresIn: '30s'})
+      if (user) return jwt.sign(`${input}_${new Date().getTime()}`, process.env.JWT_SECRET)
       else throw new Error('Customer not found!')
     },
     async verifyCustomerPaymentId(_, {input}, {loggedInUser}) {
       _auth(loggedInUser)
-      return jwt.verify(input, process.env.JWT_SECRET, async (err, jwtDecode) => {
-        if (err) throw new Error(err.message)
-        else return (await Customer.findById(jwtDecode.id)) || new Error('Customer not found!')
-      })
+      let jwtDecode = jwt
+        .verify(input, process.env.JWT_SECRET)
```

Comment:
verify function is not only checking for expiry, it checks for other jwt issue also, like jwt malformed ...
---> so inside verify, still need to handke err in callback function